event-stream-service/charts/event-stream-service/values.yaml

```
@@ -0,0 +1,252 @@
+                  "$JS.API.STREAM.MSG.GET.CHEFS",
+                  "$JS.API.STREAM.INFO.CHEFS",
+                  "$JS.API.STREAM.LIST",
+                  "$JS.API.STREAM.NAMES",
+                  "$JS.ACK.CHEFS",
+                  "$JS.ACK.CHEFS.>",
+                ]
+              subscribe: ["PUBLIC.forms.>", "PRIVATE.forms.>", "_INBOX.>"]
+          - user: "admin"
+            password: << $ADMIN_PWD >>
+            permissions:
+              publish: [">"]
+              subscribe: [">"]
+          - user: "chefs"
+            password: << $CHEFS_PWD >>
+            permissions:
+              publish:
+                [
+                  "$JS.API.INFO",
+                  "$JS.API.STREAM.CREATE.CHEFS",
+                  "$JS.API.STREAM.UPDATE.CHEFS",
+                  "$JS.API.STREAM.DELETE.CHEFS",
+                  "$JS.API.STREAM.INFO.CHEFS",
+                  "$JS.API.STREAM.PURGE.CHEFS",
+                  "$JS.API.STREAM.LIST",
+                  "$JS.API.STREAM.NAMES",
+                  "$JS.API.STREAM.MSG.DELETE.CHEFS",
+                  "$JS.API.STREAM.MSG.GET.CHEFS",
+                  "$JS.API.STREAM.SNAPSHOT.CHEFS",
+                  "$JS.API.STREAM.RESTORE.CHEFS",
+                  "$JS.API.CONSUMER.CREATE.CHEFS",
+                  "$JS.API.CONSUMER.CREATE.CHEFS.>",
+                  "$JS.API.CONSUMER.DURABLE.CREATE.CHEFS.>",
+                  "$JS.API.CONSUMER.DELETE.CHEFS.>",
+                  "$JS.API.CONSUMER.INFO.CHEFS.>",
+                  "$JS.API.CONSUMER.LIST.CHEFS",
+                  "$JS.API.CONSUMER.NAMES.CHEFS",
+                  "$JS.API.CONSUMER.MSG.NEXT.CHEFS.>",
+                  "$JS.API.CONSUMER.MSG.NEXT.CHEFS.NEW",
+                  "$JS.API.STREAM.MSG.GET.CHEFS",
+                  "$JS.ACK.CHEFS.>",
+                  "PUBLIC.forms.>",
+                  "PRIVATE.forms.>",
+                ]
+              subscribe: ["_INBOX.>"]
+  container:
+    env:
+      SYSADMIN_PWD:
+        valueFrom:
+          secretKeyRef:
+            name: ess-nats-auth
+            key: sysadmin_pwd
+      ADMIN_PWD:
+        valueFrom:
+          secretKeyRef:
+            name: ess-nats-auth
+            key: admin_pwd
+      CHEFS_PWD:
+        valueFrom:
+          secretKeyRef:
+            name: ess-nats-auth
+            key: chefs_pwd
+      CHEFSCONSUMER_PWD:
+        valueFrom:
+          secretKeyRef:
+            name: ess-nats-auth
+            key: chefsconsumer_pwd
+    merge:
+      # Each pod combines these resources with the reloader (2 containers, 1 pod)
+      # so add the below to the reloader for total resources per pod.
+      resources:
+        limits:
+          cpu: 500m
+          memory: 256Mi
+        requests:
+          cpu: 10m
+          memory: 32Mi
+  reloader:
+    merge:
+      resources:
+        limits:
+          cpu: 50m
+          memory: 64Mi
+        requests:
+          cpu: 10m
+          memory: 32Mi
```

We should be able to drop these:
```suggestion
          cpu: 20m
          memory: 12Mi
        requests:
          cpu: 10m
          memory: 8Mi
```